es/guia-para-autores.md

```diff
@@ -16,11 +16,11 @@ original: author-guidelines
 
 Si quieres traducir una lección, tienes una idea para una lección nueva o ya has escrito un tutorial que crees que puede adaptarse a *The Programming Historian en español*, contacta con [Antonio Rojas Castro]. Cuanto antes te pongas en contacto con nosotros, mucho mejor; de esta manera, te ayudaremos a plantear adecuadamente tu contribución, teniendo en cuenta el público objetivo y el nivel de conocimientos necesarios. También te asignaremos un editor para ayudarte a resolver dudas y a desarrollar la lección de la mejor manera.
 
-**¿Qué lecciones traducir?** Si quieres traducir una lección, por favor, revisa la lista de [traducciones pendientes] y ponte en contacto con nosotros. Buscamos traducciones rigurosas, de lectura amena y que, además, tengan en cuenta el contexto de España y América Latina y los recursos disponibles en lengua española.
```

Comment:
If you look at the bottom of this markdown file, I think the `[traducciones pendientes]` link is already specified there. You should not need to specify the url in parentheses after it. And the link seems to work already on the live site?

Reply:
Ok, I'll fix them there. Sorry.